app/routes/__frontend/resources/virtual-coffee/developer-resources.mdx

```diff
@@ -0,0 +1,69 @@
+---
+meta:
+  title: Resources For Developers
+  description: A compilation of various resources for developers.
+hero:
+  Hero: UndrawCollection
+order: 5
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+## Hi friends 👋
+
+<LeadText>
+
+We received various questions and resources from our members every day. Some topics arose multiple times, and others were too valuable not to share.
+
+So, we are compiling resources dropped by our members in the Slack channels and other resources that we believe would be useful for you!
```

Comment:
[Nitpick] Maybe you can hyperlink the Slack channels here @adiati98? 
```suggestion
So, we are compiling resources dropped by our members in the Slack channels and other resources that we believe would be useful for you!
```

Reply:
@CBID2 I know that we're currently postponing, but also wanted to say that bc the slack is invite-only and the resources are for the wider tech community, we wouldn't want to hyperlink to the channels.